static/app/components/core/badge/tag.chonk.tsx

```diff
@@ -0,0 +1,90 @@
+import type {DO_NOT_USE_ChonkTheme} from '@emotion/react';
+
+import type {TagProps} from 'sentry/components/core/badge/tag';
+import {space} from 'sentry/styles/space';
+import {chonkStyled} from 'sentry/utils/theme/theme.chonk';
+import {unreachable} from 'sentry/utils/unreachable';
+
+type TagType = 'default' | 'info' | 'success' | 'warning' | 'danger' | 'promotion';
+
+interface ChonkTagProps extends Omit<TagProps, 'type'> {
+  type?: TagType;
+}
+
+const LegacyMapping: Partial<Record<NonNullable<TagProps['type']>, TagType>> = {
```

Comment:
I think `PascalCasing` is reserved for `ReactComponents`, right?

```diff
- const LegacyMapping
+ const legacyMapping
```

Reply:
Yup, changed it!